awx/main/utils/ansible.py

```diff
@@ -64,6 +64,7 @@ def could_be_playbook(project_path, dir_path, filename):
                 matched = True
                 break
     except IOError:
+        logger.exception(f'failed to open {playbook_path}')
```

Comment:
![image](https://user-images.githubusercontent.com/214912/79880450-076f9700-83be-11ea-8dc4-e85418aa1710.png)

Reply:
Can you test this with a directory loop instead of a file loop?

Reply:
Yep, on it.

Reply:
![image](https://user-images.githubusercontent.com/214912/79890343-5bce4300-83cd-11ea-8016-76a642ada672.png)

![image](https://user-images.githubusercontent.com/214912/79890371-64267e00-83cd-11ea-847c-badea981f3d2.png)